CMakeLists.txt

```diff
@@ -366,6 +366,9 @@ foreach(testname ${openPMD_TEST_NAMES})
     if(TARGET Boost::unit_test_framework)
         target_link_libraries(${testname}Tests PRIVATE Boost::unit_test_framework)
     endif()
+    if(APPLE)
+        target_compile_definitions(${testname}Tests PRIVATE "-DBOOST_TEST_DYN_LINK")
```

Comment:
[wild guess](https://stackoverflow.com/questions/39171467/there-is-no-argument-provided-for-parameter-color-output-with-boost-test-and-cte), did not work.
remove

Reply:
BTW, or, `target_compile_definitions(${testname}Tests PRIVATE $<$<CXX_COMPILER_ID:AppleClang>:BOOST_TEST_DYN_LINK>)`

Reply:
Thanks, yes that's more compact via a generator expression.
We migrated to Catch2 for testing now #110, and added OSX CI in #115 .